### ISSUE TITLE
[Feature] Option not to delete previously created labels in napari-cellfinder

### ISSUE THREAD

**Issue by Eddymorphling** [brainglobe/cellfinder#473]
Hi folks,
The current napari-cellfinder >> curation workflows ends up deleting previously saved labels if new ones are being saved into the same folder. This makes it cumbersome when creating labels from multiple datasets, as I would have to create multiple yaml files. Is there an way to modify this so that I can save all training labels into a single folder, while previously created labels do not get deleted?

**Comment by adamltyson** [comment]
Hi @Eddymorphling this was designed this way originally as some users would accidentality delete existing training data, potentially undoing hours of work.

If this is causing issues we can look at changing the behaviour. I'm curious though, is it that onerous to have multiple yaml files (or to combine them into one, which is what I usually do)?

**Comment by Eddymorphling** [comment]
@adamltyson  Just another small note - Currently a self trained keras model is getting save as "model.keras" but I believe this should be named "model.keras.h5" as the brainmapper CLI tools looks for a .h5 suffix. I ended up renaming the self trained model to work.
